test/integration/scenarios/basic-setup/basic-setup.test.js

```diff
@@ -0,0 +1,47 @@
+import ScenarioUtils from '../../utils/ScenarioUtils';
+
+const path = require('path');
+
+describe('A basic karma-webpack setup', () => {
+  let scenario;
+
+  const TEST_PATH = path.resolve(__dirname, './index.scenario.js');
+
+  const config = {
+    frameworks: ['webpack', 'mocha'],
+    files: [{ pattern: TEST_PATH }],
+    preprocessors: { [TEST_PATH]: ['webpack'] },
+    webpack: {},
+    browsers: ['ChromeHeadless'],
+    // Explicitly turn off reporters so the simulated test results are not confused with the actual results.
+    reporters: [],
+    port: 2389,
+    logLevel: 'OFF',
```

Comment:
Can this be the reason, why you don't see logs?

Reply:
This was added right at the end to keep the tests from being noisy, I'll try to get a repro up sometime today.

Reply:
Looking at this made me realize that I totally missed pushing one of my files though 🤦‍♂️